### ISSUE TITLE
Empty code block rule flags classes that only contain properties

### ISSUE THREAD

**Issue by movedoa** [Riverside-Software/sonar-openedge#669]
Since 2.4.0

```
CLASS Foo FINAL:   
   DEFINE PUBLIC PROPERTY FooProp AS CHARACTER NO-UNDO
      PUBLIC GET.
      PUBLIC SET.
END CLASS.
```

**Comment by gquerret** [comment]
Reproduced

**Comment by gquerret** [comment]
In fact, there's no constructor, no method, no comment. Documentation will be updated to reflect the fact that "define" statements are not considered executable statements anymore, but I won't add an exception in the code for that. You can mark the issue as a false positive directly in SQ.
